pkg/ingress/controller/controller.go

```diff
@@ -254,7 +259,7 @@ func (c *Controller) run(ctx context.Context) {
 	}
 
 	c.goAttach(func() {
-		c.epInformer.Run(ctx.Done())
+		c.endpointsController.getInformer().Run(ctx.Done())
```

Comment:
I'm not sure whether I have the correct understanding of this `endpointsController`, but seems it doesn't have big advantages here to abstract it except we can write less `if` and `else` branches.